js/interface.js

```diff
@@ -37,7 +37,9 @@ var defaultAccessRules = [
   { type: ['delete'], allow: 'all' }
 ];
 
-var getApps = Fliplet.Apps.get();
+var getApps = Fliplet.Apps.get().then(function (apps) {
+  return _.sortBy(apps, 'name');
```

Comment:
To avoid this from happening:

- App
- Zapp
- app
...

```suggestion
  return _.sortBy(apps, function (app) {
    return app.name.toLowerCase();
  });
```

Reply:
thanks, done